### ISSUE TITLE
[SelectableTile]: selected state almost not distinguishable when in focus

### ISSUE THREAD

**Issue by janhassel** [carbon-design-system/carbon#5174]
## Detailed description

> What version of the Carbon Design System are you using?
`@carbon-components@10.9.2`

When using the keyboard to navigate between tiles (selectable and radio), the blue focus outline covers the black border which shows if a tile is selected or not. The only other distinction is the checkmark changing colors - although this is almost not noticable.

## Steps to reproduce the issue

1. Open http://react.carbondesignsystem.com/?path=/story/tile--multi-select
2. Click on the first tile to set keyboard focus in frame
3. Press <kbd>⇥ Tab</kbd>
4. With the second tile in focus, press space to select and unselect

## Additional information

![carbon-tile-focus](https://user-images.githubusercontent.com/28265588/73074548-2c856400-3eba-11ea-861a-361f8e5a1e31.gif)

In this screen recording I press the following keys:
1. <kbd>⇥ Tab</kbd>
2. <kbd>⇥ Tab</kbd>
3. <kbd>⇧ Shift</kbd> <kbd>⇥ Tab</kbd>
4. <kbd>Space</kbd>
5. <kbd>Space</kbd>
6. <kbd>⇧ Shift</kbd> <kbd>⇥ Tab</kbd>
7. <kbd>Space</kbd>
8. <kbd>⇥ Tab</kbd>
9. <kbd>Space</kbd>
10. <kbd>⇥ Tab</kbd>
11. <kbd>Space</kbd>

**Comment by laurenmrice** [comment]
For the hover state: proposing we remove the icon and only have the background color change as the indicator? This would resolve the issue we are having above with theses states not looking different enough when a focus is applied^. The selected state would still receive the icon and border change.

![Screen Shot 2020-02-04 at 10 08 06 AM](https://user-images.githubusercontent.com/43969356/73763122-9f060780-4736-11ea-9aeb-ad87649aaeaf.png)

**Comment by dakahn** [comment]
@laurenmrice that's sounds good to me

**Comment by laurenmrice** [comment]
Alright lets do it ! We also will need to remove this hover state icon from the structured list component.
